runtime/src/lib.rs

```diff
@@ -116,17 +71,13 @@ impl_opaque_keys! {
 
 /// Runtime version.
 pub const VERSION: RuntimeVersion = RuntimeVersion {
-    spec_name: create_runtime_str!("centrifuge-chain"),
-    impl_name: create_runtime_str!("centrifuge-chain"),
-    authoring_version: 10,
-    // Per convention: if the runtime behavior changes, increment spec_version
-    // and set impl_version to 0. If only runtime
-    // implementation changes and behavior does not, then leave spec_version as
-    // is and increment impl_version.
-    spec_version: 1000,
-    impl_version: 0,
+    spec_name: create_runtime_str!("altair"),
```

Comment:
We will need to add explicit types for this runtime in polkadot-js/api/apps regardless. Should we start with 1000? https://github.com/polkadot-js/api/blob/f97a6d041827e64c96e6274bc1e2c3ea275fcd5f/packages/types-known/src/spec/centrifuge-chain.ts#L72-L90

We should do something like this on the polka-js side: https://github.com/AcalaNetwork/acala.js/tree/master/packages/type-definitions/src/spec

Reply:
for altair, we would need a different one but regarding the spec_version, I have no strong opinions about any number.